Checks for string equality to fix nltk.util.escape_xml doctest

alvations opened the pull request:
Fixing #1577 by checking using string literals to check for equality.

alvations committed:
Merge pull request #63 from nltk/develop

Syncing with bleeding edge

alvations committed:
Fixing #1577 using str literals

stevenbird commented:
Thanks @alvations.
What do you think about also demonstrating the use of `xml_escape` itself, in the docstring for `xml_escape`?

I think the current documentation for `escape` is there to justify the need for `xml_escape` (and this justification could go into a Python comment before the function). Users expect to see usage examples in docstrings.

alvations commented:
@stevenbird sorry I was a little busy for the past few days. I missed out the doctest for xml_escape. Sorry about it. Added in the PR #1592 and also corrected one of the escape symbol.
